packages/medusa/src/loaders/feature-flags/sales-channels.ts

```diff
@@ -0,0 +1,10 @@
+import { FlagSettings } from "../../types/feature-flags"
+
+const SalesChannelFeatureFlag: FlagSettings = {
+  key: "sales-channels",
```

Comment:
nit: by convention let's choose to always snake_case the flag keys.
```suggestion
  key: "sales_channels",
```